extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -254,7 +255,18 @@ public void initialize() {
     private synchronized void onUpdate() {
         if (bridge != null) {
             if (pollingJob == null || pollingJob.isCancelled()) {
-                pollingJob = scheduler.scheduleAtFixedRate(pollingRunnable, 1, POLLING_FREQUENCY, TimeUnit.SECONDS);
+            	int pollingInterval = DEFAULT_POLLING_INTERVAL;
+            	try {
+            		Object pollingIntervalConfig = getConfig().get(POLLING_INTERVAL);
+            		if (pollingIntervalConfig != null) {
+            			pollingInterval = ((BigDecimal) pollingIntervalConfig).intValue(); 
+            		} else {
+            			logger.info("Polling interval not configured for this hue bridge. Using default value: {}s", pollingInterval);
+            		}
+            	} catch(NumberFormatException ex) {
+            		logger.info("Wrong configuration value for polling interval. Using default value: {}s", pollingInterval);
+            	}
+                pollingJob = scheduler.scheduleAtFixedRate(pollingRunnable, 1, pollingInterval, TimeUnit.SECONDS);
```

Comment:
If there one additional space, does Github this visualization wrong or do I need more coffee?
If there is one additional space, then the save action is not working reliable.